internal/controller/supervisorstorage/garbage_collector_test.go

```diff
@@ -381,3 +393,17 @@ func TestGarbageCollectorControllerSync(t *testing.T) {
 		})
 	}, spec.Parallel(), spec.Report(report.Terminal{}))
 }
+
+type testQueue struct {
+	called   bool
+	key      controllerlib.Key
+	duration time.Duration
+
+	controllerlib.Queue // panic if any other methods called
+}
+
+func (q *testQueue) AddAfter(key controllerlib.Key, duration time.Duration) {
+	q.called = true
```

Comment:
Is it worth testing that this was called exactly once?

Reply:
Fixed.